app/models/metric/capture.rb

```diff
@@ -13,28 +13,27 @@ def self.capture_cols
   end
 
   def self.historical_days
-    (Settings.performance.history.initial_capture_days || 7).to_i
+    Settings.performance.history.initial_capture_days.to_i
   end
 
   def self.historical_start_time
     historical_days.days.ago.utc.beginning_of_day
   end
 
   def self.concurrent_requests(interval_name)
-    requests = Settings.performance.concurrent_requests[interval_name]
-    requests ||= interval_name == 'realtime' ? 20 : 1
-    requests = 20 if requests < 20 && interval_name == 'realtime'
-    requests
+    ::Settings.performance.concurrent_requests[interval_name]
```

Comment:
current code protect against value less than 20 and we are thinking of keeping that and  adding logic to protect against values greater than 20, right ?